style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  font-family: 'Poppins', sans-serif;
+  list-style: none;
+  text-decoration: none;
+}
+
+header {
+  color: #6070ff;
+  font-size: 18px;
+  font-weight: 600;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 3%;
+  align-items: center;
+}
+
+.logo {
+  color: #6070ff;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  background-image: url(./images/mobile_bg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 98%;
+  height: 87vh;
+  border-radius: 0 0 0 100px;
+  border-bottom: 2px solid #cfd8dc;
+}
+
+.headline-section {
+  margin: 20px;
+}
+
+.headline-section h1 {
+  font-weight: 600;
+  font-size: 40px;
+  line-height: 42px;
+  color: #172b4d;
+
+  /* display: flex;
+    align-items: center; */
```

Comment:
- [ ] Kindly remove this commented code because it will make your project look messy. If you need to review a previous version of your code, you can always use git history.